test/test_helper.rb

```diff
@@ -3,7 +3,34 @@
 require 'rails/test_help'
 
 class ActiveSupport::TestCase
-  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
 
-  # Add more helper methods to be used by all tests here...
+  VCR.configure do |config|
+    config.cassette_library_dir = "fixtures/vcr_cassettes"
+    config.hook_into :webmock
+  end
+
+  def setup
+    DatabaseCleaner.start
+    Capybara.app = FeedBee::Application
+    stub_omniauth
+  end
+
+  def teardown
+    DatabaseCleaner.clean
+  end
+
+  def stub_omniauth
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+      provider: 'facebook',
+      uid: '123',
+      info: {
+        name: "Ryan Asensio"
+      },
+      credentials: {
+        oauth_token: "pizza",
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.